### ISSUE TITLE
Source going away

### ISSUE THREAD

**Issue by homerhanumat** [beanumber/imdb#7]
`ftp://ftp.fu-berlin.de/pub/misc/movies/database/README` says that the source location has changed and will go away after December 28.

**Comment by beanumber** [comment]
Yes, I saw that. They are working on something feverishly over at (https://github.com/alberanid/imdbpy). I'm waiting to see what the outcome is.

**Comment by pmarkun** [comment]
Any hints on how to get this working? :)

**Comment by beanumber** [comment]
This may be a lot easier now. Check out (https://github.com/alberanid/imdbpy/blob/master/docs/README.s3.rst)

I won't have the cycles to work on this until the summer. :(

**Comment by beanumber** [comment]
https://datasets.imdbws.com/
